images/hyperkube/Dockerfile.rhel

```diff
@@ -1,6 +1,10 @@
 FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.13 AS builder
 WORKDIR /go/src/github.com/openshift/origin
 COPY . .
+# apply a patch to net/http2 library that fixes the graceful termination of active connections, more details at: https://github.com/golang/go/issues/39776
+# this patch was prepared based on https://go-review.googlesource.com/c/go/+/240278/. The original patch is targeting the master branch and didn't apply to 1.13 version
+RUN patch -d /usr/local/go -p1 < images/hyperkube/4d8de76-1.13.patch
```

Comment:
Patching is this way seem a bit fragile given the builder images are not .z versioned. I personally think it would be better to fork the net package, patch it, vendor it back and add a simple patch for the server creation.

Reply:
> the builder images are not .z versioned

This means what?